### ISSUE TITLE
Cannot reconstruct CIFAR10 with DDIM

### ISSUE THREAD

**Issue by zzz313** [lucidrains/denoising-diffusion-pytorch#219]
Hi,

I have trained the DDPM for CIFAR10 with two settings:  1. T= 500 and epoch = 500.  2. T = 1000 and epoch = 200. 

Both models can generate CIFAR10 images, but I failed at reconstructing the CIFAR10 with the DDIM process, it is still a random generation, any advice???  And the loss fluctuates around 0.0015 after 100 epochs.  

The image scale is 64x64.
